ThirdParty/cronosplay/include/defi-wallet-core-cpp/src/contract.rs.cc

```diff
@@ -1,6 +1,3 @@
-#pragma warning(disable : 4583)
-#pragma warning(disable : 4582)
```

Comment:
no need for these pragmas anymore?

Reply:
plus in other generated code

Reply:
because we're using static library, not used now